src/itdelatrisu/opsu/Options.java

```diff
@@ -852,6 +858,24 @@ public static File getReplayDir() {
 	}
 
 	/**
+	 * Returns the horizontal Resolution.
+	 * If no game has been started then a default value is returned.
+	 * @return the horizontal resolution of the latest game instance to be started.
+	 */
+	public static int getLatestResolutionWidth() {
```

Comment:
What's the purpose of these as opposed to `container.getWidth()` or `Display.getWidth()`?